### DISCUSSION TITLE
Production Grade Network Tools Waterfall

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#675] [General]
@philipbaileynar started [this flowchart](https://docs.google.com/drawings/d/14j-it_eIef5K1INQy6hgFm_91hJpzadFD1Wca0Wrpbg/edit?usp=sharing) to map out the dependencies of our various prouction-grade network tools. @joewheaton modified it to try and reflect the distinctions between our channel network models, our single riverscape network model (VBET), and the new class of hybrid channel & riverscape network models (i.e. those that calculate outputs both on channel AGOs (reach segments - aggregated geographic objects) and riverscape IGOs (integrated geographic objects, moving window valley bottoms). 

![Riverscapes Tools Waterfall](https://user-images.githubusercontent.com/9723905/206922706-50186d75-5aba-4401-b0cb-4028b7b4c9ce.png)

- Everything depends on Riverscapes Context. 
- VBET additionally depends on Channel Area and TauDEM
- All hybrid models depend on VBET's riverscape network (specifically DGOs and IGOs)
- GNAT is its own little metric generation engine

Comments, questions, confusion, concerns, etc? FYI @Riverscapes/developers and @Riverscapes/riverscape-analysts?

**Comment by lauren-herbine** [comment]
I added some comments on the google drawing! @joewheaton
